src/config/section/compute.js

```diff
@@ -104,7 +104,8 @@ export default {
           icon: 'reload',
           label: 'label.action.reboot.instance',
           dataView: true,
-          show: (record) => { return ['Running'].includes(record.state) }
+          show: (record) => { return ['Running'].includes(record.state) },
+          args: ['bootintobios']
```

Comment:
this args field should be conditional on `hypervisor === vmware`. will be done after #231 has been merged with its [AutoGenView.vue changes](https://github.com/apache/cloudstack-primate/pull/231/files#diff-83f2f8976f46b2444d31fcd1895d8165R548)